src/__swaps__/screens/Swap/hooks/useSwapTextStyles.ts

```diff
@@ -79,19 +60,30 @@ export function useSwapTextStyles({
   });
 
   const isInputZero = useDerivedValue(() => {
-    const isZero =
-      !internalSelectedInputAsset.value ||
-      (inputValues.value.inputAmount === 0 && inputMethod.value !== 'slider') ||
-      (inputMethod.value === 'slider' && equalWorklet(inputValues.value.inputAmount, 0));
-    return isZero;
+    const isInputAmountZero = inputValues.value.inputAmount === 0;
```

Comment:
the `isInputZero` and `isOutputZero` still unfortunately rely on distinguishing between 0 (number) and "0" (string) for what is a zero state from when a user deletes all the values (number) vs when the user explicitly typed in a 0 (string)